public/client.js

```diff
@@ -1,25 +1,36 @@
-import UIkit from 'uikit'
+// $('#email').keyup(function () {
+//   const email = $('#email')
+//   if (email.val()) {
+//     if (validateEmail(email.val())) {
+//       email.css({ 'color': '#f0506e', 'border-color': '#f0506e' })
+//     } else {
+//       email.css({ 'color': '#32d296', 'border-color': '#32d296' })
+//     }
+//   }
+// })
 
 $('#username').keyup(function () {
-  const userfield = $('#username')
-  const profile = 'https://api.github.com/users/' + userfield.val()
-
-  fetch(profile)
-    .then((response) => {
-      response.json()
-        .then((data) => {
-          if (data.message) {
-            userfield.css({ 'color': '#f0506e', 'border-color': '#f0506e' })
-            $('#errorMsg').html('Username invalid.')
-          } else {
-            userfield.css({ 'color': '#32d296', 'border-color': '#32d296' })
-            $('#errorMsg').html(' ')
-          }
-        })
-    })
-    .catch((e) => {
-      console.log(e)
-    })
+  const username = $('#username')
+  if (username.val()) {
+    // http://aashutoshrathi.glitch.me/api/gh/ to increase API calls without 403.
+    const profile = 'http://aashutoshrathi.glitch.me/api/gh/' + username.val()
+    fetch(profile)
+      .then((response) => {
+        response.json()
```

Comment:
```suggestion
        return response.json()
```
take the `then` inside to outer so that the logic is more `visible` to new-comers

Reply:
Committed few changes.
Can you be more clear about 
>take the then inside to outer so that the logic is more visible to new-comers

😅